processors/processExpand.mjs

```diff
@@ -630,6 +630,9 @@ function processExpand(entries, meta) {
     cosmetics(e) {
       expand(e);
     },
+    neutral_token(e) {
+      expand(e);
```

Comment:
Can we just save time and key to use less space? I think we do this with wards

Reply:
Each object will be within a player right? So we don't need to store the slot/player_slot etc. in the final output